spec/values/calculation/mod.hrx

```diff
@@ -0,0 +1,273 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `mod()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> error/syntax/no_arg/input.scss
+a {b: calc(mod())}
```

Comment:
Other than perhaps for a test that explicitly tests nesting (which I don't think is necessary for each individual function), these tests should just test `mod()` on its own, rather than wrapping any of them in a `calc()`.

Same for `pow()`, `rem()`, and `sign()`.

Reply:
Done :)